R/volcanoplot.R

```diff
@@ -230,10 +241,11 @@ srv_g_volcanoplot <- function(id,
 
         card$append_text(final_encodings, style = "verbatim")
         card$append_text("Plot", "header3")
-        card$append_plot(plot_r())
+        card$append_plot(plot_r(), dim = pws_p$dim())
         if (isTRUE(input$show_top_gene)) {
           card$append_text("Table", "header3")
           card$append_table(show_top_gene_diffexpr())
+          # card$append_table(show_top_gene_diffexpr(), dim = pws_t$dim())
```

Comment:
same here

Reply:
append_table method does not have the dim argument so should be removed.

Reply:
I remove the comment but be very careful as the table easily could be ver long and take to much space in the report. May be the head + tail of the table is enough?